src/index.js

```diff
@@ -227,7 +227,7 @@ class AlertsPlugin {
       };
     } else {
       throw new Error(
-        `Missing type for alarm ${alarm.name} on function ${functionName}, must be one of 'static' or 'anomalyDetection'`
+        `Missing type for alarm ${definition.name} on function ${functionName}, must be one of 'static' or 'anomalyDetection'`
```

Comment:
I also changed this to `definition.name` because I think that there was a mistake. If all previous ifs are false and we end up here, `alarm` should be undefined and therefore using `alarm.name` probably results in a "Can't read property name of undefined" error?